deploy/tests/network.ts

```diff
@@ -77,76 +81,320 @@ describe('network', () => {
+                    token: dai,
+                    whale: daiWhale
+                },
+                [TestPools.LINK]: {
+                    token: link,
+                    whale: linkWhale
+                }
+            };
+
+            for (const { token } of Object.values(pools)) {
+                await poolCollection.connect(daoMultisig).setDepositLimit(token, MAX_UINT256);
+            }
+        });
+
+        it('should perform deposits', async () => {
+            for (const [tokenSymbol, { token, whale }] of Object.entries(pools)) {
+                const isNativeToken = tokenSymbol === TestPools.ETH;
+
+                const depositAmount = toWei(1000);
+
+                for (let i = 0; i < 5; i++) {
+                    const { liquidity: prevLiquidity } = await poolCollection.poolData(token);
+
+                    if (!isNativeToken) {
+                        const tokenContract = await Contracts.ERC20.attach(token);
+                        await tokenContract.connect(whale).approve(network.address, depositAmount);
+                    }
+
+                    await network.connect(whale).deposit(token, depositAmount, {
+                        value: isNativeToken ? depositAmount : BigNumber.from(0)
+                    });
+
+                    const { liquidity } = await poolCollection.poolData(token);
+                    expect(liquidity.stakedBalance).to.equal(prevLiquidity.stakedBalance.add(depositAmount));
+
+                    expect({
+                        n: prevLiquidity.bntTradingLiquidity,
+                        d: prevLiquidity.baseTokenTradingLiquidity
+                    }).to.be.almostEqual(
+                        {
+                            n: liquidity.bntTradingLiquidity,
+                            d: liquidity.baseTokenTradingLiquidity
+                        },
+                        {
+                            maxAbsoluteError: new Decimal(0),
+                            maxRelativeError: new Decimal('0000000000000000000001')
+                        }
+                    );
+                }
+            }
+
+            const { bntWhale } = await getNamedSigners();
+
+            // perform a few BNT deposit tests
+            const bntAmount = toWei(10);
+
+            for (let i = 0; i < 5; i++) {
+                const prevBNBNTAmount = await getBalance(bntBNT, bntWhale);
+                const prevVBNTTokenAmount = await getBalance(vbnt, bntWhale);
+                const prevTotalSupply = await bnt.totalSupply();
+
+                await bnt.connect(bntWhale).approve(network.address, bntAmount);
+                await network.connect(bntWhale).deposit(bnt.address, bntAmount);
+
+                const receivedBNBNTAmount = (await getBalance(bntBNT, bntWhale)).sub(prevBNBNTAmount);
+
+                expect(receivedBNBNTAmount).be.gt(0);
+                expect(await getBalance(vbnt, bntWhale)).to.equal(prevVBNTTokenAmount.add(receivedBNBNTAmount));
```

Comment:
I'd use:
```
const currBNBNTAmount = await getBalance(bntBNT, bntWhale);

expect(currBNBNTAmount).be.gt(prevBNBNTAmount);
expect(await getBalance(vbnt, bntWhale)).to.equal(prevVBNTTokenAmount.add(currBNBNTAmount).sub(prevBNBNTAmount));
```

Reply:
The current check is explicit and I don't see how your suggestion improves on it. Moving forward.